testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimatedCubeController.cs

```diff
@@ -179,13 +179,27 @@ private void BeginAttack(int weaponType)
             m_NetworkAnimator.SetTrigger("Attack");
         }
 
+        private void SetLayerWeight(int layer, float weight)
+        {
+            m_Animator.SetLayerWeight(layer, weight);
+        }
+
+        private float GetLayerWeight(int layer)
+        {
+            return m_Animator.GetLayerWeight(layer);
+        }
+
         private void LateUpdate()
         {
 
             if (!IsSpawned || !IsOwner)
             {
                 if (!IsOwner && IsSpawned)
                 {
+                    if (Input.GetKeyDown(KeyCode.Alpha4))
```

Comment:
This looks like debug code maybe?

Reply:
It is part of the manual test that just allows you to verify that when changes are applied to a layer's weight that it was updated on other non-authoritative/owner instances.